en-US/mozorg/about/governance/policies/community-hotline.lang

```diff
@@ -0,0 +1,25 @@
+## NOTE: Demo page: https://www-dev.allizom.org/about/governance/policies/participation/
```

Comment:
Is this the correct URL for testing? I can't find the content.

Reply:
Oops, bad copy-paste. Fixed now.